Code/Tools/FBuild/FBuildCore/Graph/NodeGraph.cpp

```diff
@@ -1436,6 +1436,77 @@ Node * NodeGraph::FindNodeInternal( const AString & fullPath ) const
 	return nullptr;
 }
 
+// FindNearestNodesInternal
+//------------------------------------------------------------------------------
+size_t NodeGraph::FindNearestNodesInternal( const AString & fullPath, const Node **nodes, uint32_t capacity ) const
```

Comment:
Question: Is there a reason you don't use an Array< Node * > for this? It would track capacity and size for you as well as providing assertions for indexing.